<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>102</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>421</width>
     <height>281</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>background</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>aeroplane</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bicycle</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bird</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>boat</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bottle</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bus</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>car</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>cat</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>chair</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>cow</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>diningtable</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>dog</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>horse</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>motorbike</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>person</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>pottedplant</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sheep</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sofa</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>train</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
   <item>
    <property name="text">
     <string>tvmonitor</string>
    </property>
    <property name="checkState">
     <enum>Unchecked</enum>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
